### PR TITLE
Carousel homepage

### PR DESCRIPTION
**Pull request by darylfung96**
Added warpAround for carousel and autoPlay

resolved #115

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
+
 html {
     height: 100%;
+    font-family: gilroy-semibold;
     font-weight:normal !important;
 }
 
 body {
-    font-family: 'Raleway', sans-serif;
+    /* font-family: 'Raleway', sans-serif; */
```
Comment (jadasc) [inline assets/css/main.css]:
```suggestion

```

**Comment by jadasc** [comment]
Are there any other ways of controlling the carousel apart from using the left and right arrows, or clicking and dragging?

File: assets/css/main.css [hunk 2]
```diff
@@ -1,15 +1,37 @@
...
     height: 100%;
     font-weight:normal !important;
 }
 
 
+/* carousel */
+.carousel-cell {
+    width: 100%;
+    /* margin-right: 10px; */
```
Comment (jadasc) [inline assets/css/main.css]:
```suggestion
```

File: assets/css/main.css [hunk 3]
```diff
@@ -1,15 +1,37 @@
+@font-face {
+    font-family: gilroy-medium;
+    src: url("{{ site.baseurl }}/assets/font/Gilroy-Medium.otf") format("opentype");
```
Comment (jadasc) [inline assets/css/main.css]: It looks perfectly fine on my machine (and not to mention great!), but I noticed that the file you added as part of this PR is a `.ttf` file and not `.otf` file, as specified here 🤷‍♀

File: assets/css/main.css [hunk 4]
```diff
@@ -1,15 +1,37 @@
...
+}
+
+
 html {
     height: 100%;
+    font-family: gilroy-semibold;
```
Comment (jadasc) [inline assets/css/main.css]: Do we need to add a `@font-face` for **gilroy-semibold** or is it part of the one added above, **gilroy-medium**?

**Comment by darylfung96** [comment]
> Would it be possible to add `flickity` as a dependency? [npmjs.com/package/flickity](https://www.npmjs.com/package/flickity)

@jadasc It's possible to add flickity as dependency but I think we might need to make changes to how the server is getting setup so that the server would use npm to install all the dependencies before hosting the website

File: assets/css/main.css (hunk 3 above)
Comment (darylfung96) [reply assets/css/main.css]: that's a good catch! it's updated

File: assets/css/main.css (hunk 4 above)
Comment (darylfung96) [reply assets/css/main.css]: this should be gilroy-medium, I just updated it, good catch as well

**Comment by jadasc** [comment]
@darylfung96

It might be better to install `flickity` as a dependency, rather than have the files in our repo.  That way we can keep track of versioning, as well I think it has some optimizations that will reduce the size of our build

As for the carousel control, I was just curious. Thanks for offering to look into it :)

**Pull request merged**